kubernetes/deployment.tmpl

```diff
@@ -112,6 +112,56 @@ spec:
               value: staging
             - name: REDIS_URI
               value: "redis://hamlet-staging-redis"
+          volumeMounts:  # ??? Is this necessary?
+            - name: static-assets
+              mountPath: "/static"
+
+        - name: hamlet-staging-nginx
+          image: zooniverse/apps-nginx:xenial
+          ports:
+            - containerPort: 80  # ??? could this conflict with the Django app?
+          volumeMounts:
+            - name: static-assets
+              mountPath: "/static"
```

Comment:
Update: as Cam pointed out, _it's not._ Static files will now be stored on the Volume mount at `/static-assets` instead of within the, uh, Hamlet app's `/hamlet/static` folder.